rascal-vscode-extension/.vscode/settings.json

```diff
@@ -1,11 +1,13 @@
 // Place your settings in this file to overwrite default and user settings.
 {
     "files.exclude": {
-        "out": false // set this to true to hide the "out" folder with the compiled JS files
+        "out": false, // set this to true to hide the "out" folder with the compiled JS files
+        "dist": false, // set this to true to hide the "out" folder with the compiled JS files
     },
     "search.exclude": {
-        "out": true // set this to false to include "out" folder in search results
+        "out": true, // set this to false to include "out" folder in search results
+        "dist": true, // set this to false to include "out" folder in search results
     },
     // Turn off tsc task auto detection since we have the necessary tasks as npm scripts
     "typescript.tsc.autoDetect": "off",
-}
\ No newline at end of file
+}
```

Comment:
I find these comments superfluous. the key is called "search.exclude" or "files.exclude" so that says enough.

Reply:
true, they came from the tool that generated them. I'm gonna cleanup some.